halo2_proofs/src/poly/commitment/verifier.rs

```diff
@@ -92,92 +89,89 @@ pub fn verify_proof<'a, C: CurveAffine, E: EncodedChallenge<C>, T: TranscriptRea
         let l = transcript.read_point().map_err(|_| Error::OpeningError)?;
         let r = transcript.read_point().map_err(|_| Error::OpeningError)?;
 
-        let challenge_packed = transcript.squeeze_challenge();
-        let challenge = *challenge_packed.as_challenge_scalar::<()>();
+        let u_j_packed = transcript.squeeze_challenge();
+        let u_j = *u_j_packed.as_challenge_scalar::<()>();
 
         rounds.push((
-            l,
-            r,
-            challenge,
-            /* to be inverted */ challenge,
-            challenge_packed,
+            l, r, u_j, u_j, // to be inverted
```

Comment:
Oh, now I know why I did `/* to be inverted */` originally: so that rustfmt wouldn't do this to it.

Reply:
```suggestion
            l, r, u_j, /* to be inverted */ u_j,
```